src/Storages/ColumnsDescription.cpp

```diff
@@ -177,14 +177,14 @@ static auto getNameRange(const ColumnsDescription::ColumnsContainer & columns, c
     for (; begin != columns.end(); ++begin)
     {
         if (begin->name == name_without_dot)
-            return std::make_pair(begin, std::next(begin));
+            return std::pair(begin, std::next(begin));
```

Comment:
In case someone wonders why we don't return `{begin, std::next(begin)}`: The reason is that sometimes functions have return type `auto`. In this case, we need to help the compiler and specify explicitly that a `pair` is being returned.